### ISSUE TITLE
cannot access protected  method java.lang.Class com.appsflyer.internal.AFb1mSDK$$.loadClass

### ISSUE THREAD

**Issue by rehmanilyas** [AppsFlyerSDK/appsflyer-unity-plugin#231]
Appsflyer version 6.12.10 showing this crash issue in crashlytics. 

Fatal Exception: java.lang.Error: FATAL EXCEPTION [pool-34-thread-1]
Unity version     : 2020.3.15f2
Device model      : OnePlus OnePlus8Pro
Device fingerprint: OnePlus/IN2023/OnePlus8Pro:11/QKR1.191246.002/2006701342:user/release-keys
Build Type        : Release
Scripting Backend : IL2CPP
ABI               : arm64-v8a
Strip Engine Code : true

```
Caused by java.lang.Error: java.lang.IllegalAccessException: Class java.lang.Class<java.lang.reflect.Method> cannot access protected  method java.lang.Class com.appsflyer.internal.AFb1mSDK$$.loadClass(java.lang.String, boolean) of class java.lang.Class<com.appsflyer.internal.AFb1mSDK$$>
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1173)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
       at java.lang.Thread.run(Thread.java:923)

Caused by java.lang.IllegalAccessException: Class java.lang.Class<java.lang.reflect.Method> cannot access protected  method java.lang.Class com.appsflyer.internal.AFb1mSDK$$.loadClass(java.lang.String, boolean) of class java.lang.Class<com.appsflyer.internal.AFb1mSDK$$>
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:423)
       at com.appsflyer.internal.AFb1mSDK$$.AFKeystoreWrapper(:100)
       at com.appsflyer.internal.AFb1mSDK$$.values(:19)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:423)
       at com.appsflyer.internal.AFb1pSDK.values(:125)
       at com.appsflyer.internal.AFd1eSDK.values(:10)
       at com.appsflyer.internal.AFd1hSDK.values(:1)
       at com.appsflyer.internal.AFd1eSDK.AFKeystoreWrapper(:1)
       at com.appsflyer.internal.AFd1nSDK.AFInAppEventParameterName(:6)
       at com.appsflyer.internal.AFd1tSDK$5.run(:27)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
       at java.lang.Thread.run(Thread.java:923)
```

**Comment by af-fess** [comment]
Possible duplicate of https://github.com/AppsFlyerSDK/appsflyer-android-sdk/issues/48

**Comment by af-do** [comment]
Hi @rehmanilyas,
As this looks like an issue in the underlying Android SDK and was fixed in 6.12.2 which is included in the Unity 6.12.20 version that was recently released - I'm closing this issue, please re-open it if you will encounter this issue once again in 6.12.20, or continue the discussion in Android SDK issue linked above.
